src/views/catalog/EnableInstanceTypeTechPreviewModal/EnableInstanceTypeTechPreviewModal.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import { Trans } from 'react-i18next';
+import instanceTypeTechPreviewModalImage from 'images/instance-type-tech-preview-modal-img.svg';
+
+import { useIsAdmin } from '@kubevirt-utils/hooks/useIsAdmin';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { usePreviewFeatures } from '@kubevirt-utils/hooks/usePreviewFeatures/usePreviewFeatures';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import {
+  Button,
+  ButtonVariant,
+  Modal,
+  ModalVariant,
+  Split,
+  SplitItem,
+  Stack,
+  StackItem,
+  Text,
+  TextVariants,
+  Title,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+type EnableInstanceTypeTechPreviewModalProps = {
+  navigateToCatalog: () => void;
+};
+
+const EnableInstanceTypeTechPreviewModal: FC<EnableInstanceTypeTechPreviewModalProps> = ({
+  navigateToCatalog,
+}) => {
+  const { t } = useKubevirtTranslation();
+
+  const { instanceTypesEnabled, loading, toggleInstanceTypesFeature, error } = usePreviewFeatures();
+
+  const isAdmin = useIsAdmin();
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  useEffect(() => {
+    if (!isEmpty(error)) {
+      setIsOpen(true);
+      return;
+    }
+    if (!loading) {
+      setIsOpen(!instanceTypesEnabled);
+    }
+  }, [loading, instanceTypesEnabled, error]);
+
+  const onClose = useCallback(() => {
+    setIsOpen(false);
+    navigateToCatalog();
+  }, [navigateToCatalog]);
+
+  return (
+    <Modal
+      header={
+        <Title headingLevel="h1">{t('Creating a VirtualMachine from an InstanceType')}</Title>
```

Comment:
I think that "Creating **a** VirtualMachine" or "an InstanceType" is not very consistent with the rest of the text of our UI. But I've better commented the design doc, as this is more for a discussion with the UX/doc team. Maybe this is gonna change in the future 😇 😅

Reply:
responded in the doc :)